### ISSUE TITLE
Allow opening GWZ files in addition to GWC

### ISSUE THREAD

**Issue by jakedot** [cgeo/WhereYouGo#49]
When developing Wherigos, using GWZ files can be easier to handle than GWC, e.g. you can open them with ZIP Editor and quickly change resources. Applications like webwigo.net allow opening GWZ, so it would be nice to have in WhereYouGo as well.

Extracted from findings in #40, this isn't supported yet, please correct me if I'm wrong and just made a mistake when testing this.

**Comment by SchoolGuy** [comment]
I don't see this needed in WhereYouGo. This is only needed if you want to develop. WhereYouGo is a Wherigo Player and not a development tool. Because a GWZ is useless for a wherigo-player if you are not compiling it first to my knowledge.

**Comment by jakedot** [comment]
By `wherigo-player`, do you mean the person playing or the application? If the person, yes, absolutely true. The application, well that depends on whether it implements the capability, a catch-22.

**Comment by SchoolGuy** [comment]
Sorry for not clarifying but I meant the application. I don't see WhereYouGo becoming a full fledged Wherigo Debugging & Developing Tool currently.

**Comment by jakedot** [comment]
There's really no need to take it that far. What I'm remembering, the GWZ
format isn't all that different from GWC, it just contains the LUA code in
text form instead of compiled, and shouldn't be too difficult to run
either. It may just make it easier to test on real devices  for developers.
But I agree that it's a low priority.

**Comment by SchoolGuy** [comment]
Well if you look at the gwc format there is a littlebit of extra Metadata which is not available in a gwz (unlock code, player name,...). I think it would be nice to have but let's clean up WhereYouGo before we start doing new features.

**Comment by jakedot** [comment]
Closing for now.
